src/leap/bitmask/gui/logwindow.py

```diff
@@ -72,36 +132,45 @@ def __init__(self, parent, handler):
         self._current_filter = ""
         self._current_history = ""
 
-        # Load logging history and connect logger with the widget
-        self._logging_handler = handler
-        self._connect_to_handler()
+        self._set_logs_to_display()
+
+        self._my_handler = QtLogHandler(format_string=LOG_FORMAT)
+
         self._load_history()
+        self._connect_to_handler()
```

Comment:
Both logbook and QT have the concept of handlers, is that correct? Maybe we could change this method's name to `_connect_to_qt` so we avoid mentioning handlers here?